doc/whats_new.rst

```diff
@@ -66,7 +58,7 @@ Bug fixes
 
    - Fix issue where ``min_grad_norm`` and ``n_iter_without_progress``
      parameters were not being utilised by :class:`manifold.TSNE`.
-     `#6497 <https://github.com/scikit-learn/scikit-learn/pull/6497>`_
```

Comment:
I realise. I've not worked out the regex to do so, and have little time to put to it manually. Contrib welcome.